migration/schema/v1/tables.go

```diff
@@ -1011,3 +1011,39 @@ func ConvertV2SamlToIdpConfig(samlMap map[string]any) (v3Config *IdpConfig, err
 	v3Config.Enabled = true
 	return
 }
+
+// Group is respretation of a row in "platformdb"."groups" table
+type Group struct {
+	GroupID     string `json:"group_id,omitempty"`
```

Comment:
As a convention, I'm avoiding `omitempty` for any columns that have `NOT NULL` on them, so I'd prefer to see these removed.